### PR TITLE
chore, fix(egen): follows new template, markdown updates and fixes

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
This PR includes the following updates to the `challenger_vs_blessed_deployment_method.ipynb` notebook:

- Change REGION to LOCATION
- Remove IS_TESTING condition at cleaning step
- Adds IS_COLAB in the service account setup
- Removes USER in the installation
- Moves the deletion of local files to the cleaning up section
- Add Colab Enterprise link and other updates from new template.
- Reorganizes the heading styles to match the steps.
- Removes unnecessary code highlights
- Removes future tense 
- Contracts words like "do not" --> "don't"
- Update K80 GPU with T4 GPU

<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 1]
```diff
@@ -548,7 +476,7 @@
       "source": [
         "#### Set pre-built containers\n",
         "\n",
-        "Set the pre-built Docker container image for training and prediction.\n",
+        "Set the pre-built Docker container image for prediction.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:479]: change "pre-built" to "prebuilt"

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 2]
```diff
@@ -583,32 +511,21 @@
       "source": [
         "#### Set machine type\n",
         "\n",
-        "Next, set the machine type to use for prediction.\n",
+        "Next, set the machine type to use for deployment.\n",
         "\n",
-        "- Set the variable `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you will use for for training.\n",
-        " - `machine type`\n",
-        "     - `n1-standard`: 3.75GB of memory per vCPU.\n",
+        "- Set the variable `DEPLOY_COMPUTE` to configure  the compute resources for the VMs you use for deployment.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:516]: Remove extra space between "configure" and "the":

Set the variable `DEPLOY_COMPUTE` to configure the compute resources for the VMs you use for deployment.

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 3]
```diff
@@ -583,32 +511,21 @@
...
+        " - `machine_type`\n",
+        "     - `n1-standard`: 3.75GB of memory per vCPU\n",
         "     - `n1-highmem`: 6.5GB of memory per vCPU\n",
         "     - `n1-highcpu`: 0.9 GB of memory per vCPU\n",
         " - `vCPUs`: number of \\[2, 4, 8, 16, 32, 64, 96 \\]\n",
         "\n",
-        "*Note: The following is not supported for training:*\n",
+        "**Note**: The following are not supported.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:523]: change "are not" to "aren't"

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 4]
```diff
@@ -755,7 +685,7 @@
         "\n",
         "Next, upload the model's evaluation from the custom training job to the corresponding entry in the Vertex AI Model Registry.\n",
         "\n",
-        "Currently, there is not yet support for this method in the SDK. Instead, you use the lower level GAPIC API interface."
+        "Currently, there is no support for this method yet in the Python SDK. Instead, you use the lower level GAPIC API interface."
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:688]: change:  "Currently, there is no support for this method yet in the Python SDK"
to: " "Currently, the Vertex AI Python SDK not support for this method."

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 5]
```diff
@@ -780,13 +710,13 @@
         "id": "628de0914ba1"
       },
       "source": [
-        "## Creating an `Endpoint` resource\n",
+        "## Creating an endpoint resource\n",
         "\n",
-        "You create an `Endpoint` resource using the `Endpoint.create()` method. At a minimum, you specify the display name for the endpoint. Optionally, you can specify the project and location (region); otherwise the settings are inherited by the values you set when you initialized the Vertex AI SDK with the `init()` method.\n",
+        "You create an endpoint resource using the `Endpoint.create()` method. At a minimum, you specify the display name for the endpoint. Optionally, you can specify the project and location (region); otherwise the settings are inherited by the values you set when you initialized the Vertex AI SDK with the `init()` method.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:715]: change: Vertex AI SDK 
to: Vertex AI Python SDK

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 6]
```diff
@@ -858,17 +788,17 @@
...
         "\n",
-        "First, you define a component to upload the trained challenger model as a version to the blessed model in the `Vertex AI Model Registry`. The component takes the following arguments:\n",
+        "First, you define a component to upload the trained challenger model as a version to the blessed model in the Vertex AI Model Registry. The component takes the following arguments:\n",
         "\n",
         "- `parent_model`: The full resource name of the blessed model.\n",
         "- `artifact_uri`: The Cloud Storage location of the model artifacts for the challenger model.\n",
         "- `serving_container`: The serving container for the challenger model.\n",
         "- `project`: Your Project ID.\n",
-        "- `region`: Your region."
+        "- `region`: The region where you want to create/use the resources."
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:801]: change: "create/use"
to: "create or use"

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 7]
```diff
@@ -914,10 +844,10 @@
         "\n",
         "Next, you define a component to import the evaluation metrics for the challenger model to the Model Registry. The component takes the following arguments:\n",
         "\n",
-        "- `display_name`: Human readable name for the evaluation metrics\n",
+        "- `display_name`: Human readable name for the evaluation metrics.\n",
         "- `metrics`: The evaluation metrics formatted for classification.\n",
         "- `parent_model_resource`: The full resource name for the challenger model version.\n",
-        "- `region`: The region."
+        "- `region`: The region where you want to create/use the resources."
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:850]: change: "create/use"
to: "create or use"

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 8]
```diff
@@ -1170,11 +1106,11 @@
         "id": "e1d089d1d0d9"
       },
       "source": [
-        "### Get the latest state of the production endpoint\n",
+        "## Get the latest state of the production endpoint\n",
         "\n",
-        "Now that the pipeline has finished, the challenger (version 2) model has replaced the previous blessed model on the production endpoint.\n",
+        "Once your pipeline execution is finished, you can notice that the challenger (version 2) model has replaced the previous blessed model on the production endpoint.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:1111]: change: "Once your pipeline execution is finished, you can notice that the challenger (version 2) model has replaced the previous blessed model on the production endpoint"

to: "Once your pipeline execution is finished, notice that the challenger (version 2) model has replaced the previous blessed model on the production endpoint"

File: notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb [hunk 9]
```diff
@@ -1170,11 +1106,11 @@
...
         "\n",
-        "Next your display the latest information on the deployed models for the production endpoint, and then display the traffic split. The resource ID for the 100% entry is the resource ID for the challenger (v2) model."
+        "Now, display the latest details of the deployed models from the production endpoint, and then display the traffic split. Notice that the resource ID for the 100% entry is the resource ID for the challenger (v2) model."
```
Comment (kittyabs) [inline notebooks/official/pipelines/challenger_vs_blessed_deployment_method.ipynb:1113]: change: " Notice that the resource ID" 

to:  "Notice the resource ID" (remove extra "that")

**Review by kittyabs — COMMENTED** [review]
numerous edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**